poe-tasks/gradle-connector-tasks.toml

```
@@ -57,6 +57,26 @@ else
 fi
 '''
 
+# Identical to test-integration-tests except excludes tests marked with @Tag("requires-creds")
```

What about just making this the help text? Same readability benefit for a reader of the code, but also prints to the end user when commands are listed.

```suggestion
test-integration-tests-no-creds.help = """Identical to 'test-integration-tests' except excluding tests marked with '@Tag("requires-creds")'."""
```